pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -75,6 +77,24 @@ func makeSegmentstorePodSpec(p *api.PravegaCluster) corev1.PodSpec {
 			},
 		},
 	}
+	if len(configMapName) > 0 {
```

Comment:
Why a length check, what if user enters a name that is 1 blank space? Could we do a string.trim() and then check for empty string "" ?